src/App.js

```diff
@@ -1,15 +1,12 @@
 import React, {Component} from "react";
```

Comment:
From react 17.0.0, `import react from 'React'` is no longer needed for components to render

(Of course, named imports like `useState` are still nessecary)

Reply:
Throws an error

Reply:
Yep, my bad.
Add these to your `eslint.json` file:
```
{
  // ...
  "rules": {
    // ...
    "react/jsx-uses-react": "off",
    "react/react-in-jsx-scope": "off"
  }
}
```